aria-practices.html

```diff
@@ -3734,6 +3734,390 @@ <h5>Examples</h5>
+
+        <p>
+          If a contact changes their display name, the text change would also be announced:
+          "Alice has changed their name to 4liz".
+          After a timeout, the text "Alice has changed their name to " could be removed, without causing a new announcement.
+        </p>
+      </section>
+
+      <section id="triggering_live_regions">
+        <h4>Triggering Live Regions</h4>
+
+        <p>
+          Additions and removals in the accessibility tree can happen due to changes to the DOM tree or changes to the applied CSS.
+          For example, changing the CSS <code>display</code> property to <code>none</code> causes the element to be removed from the accessibility tree.
+          See the <a href="#accessibility-tree">Accessibility tree</a> section for more details.
+        </p>
+      </section>
+    </section>
+
+    <section id="special_case_live_regions">
+      <h3>Special Case Live Regions</h3>
+
+      <p>
+        The roles listed below implicitly set the <code>aria-live</code> attribute to indicate that it is a live region.
+        When using these roles, the <code>aria-live</code> attribute can be omitted,
+        or it can be specified to change the value from the default.
+      </p>
+
+      <ul>
+        <li><code>alert</code></li>
+        <li><code>log</code></li>
+        <li><code>status</code></li>
+        <li><code>timer</code></li>
+        <li><code>marquee</code></li>
+      </ul>
+
+      <section id="live_region_role_alert">
+        <h4>Live Region Role <code>alert</code></h4>
+
+        <p>
+          The <code>alert</code> role indicates important, usually time-sensitive, information.
+          Use this role when focus is not moved to the message, and the user is not expected to close the message.
+          For an alert dialog that can be closed, the the <code>alertdialog</code> role instead.
+        </p>
+
+        <p>
+          The default value for <code>aria-live</code> is <code>assertive</code>.
+          The default value for <code>aria-atomic</code> is <code>true</code>.
+        </p>
+
+        <p>
+          See the <a href="#alert">Alert design pattern</a> and the related <a href="https://w3c.github.io/aria-practices/examples/alert/alert.html">Alert Example</a>.
+        </p>
+      </section>
+
+      <section id="live_region_role_log">
+        <h4>Live Region Role <code>log</code></h4>
+
+        <p>
+          The <code>log</code> role indicates that new information is added in meaningful order and old information might disappear.
+        </p>
+
+        <p>
+          The default value for <code>aria-live</code> is <code>polite</code>.
+        </p>
+
+        <p>
+          For example, a chat log would be an appropriate use case for the <code>log</code> role.
+        </p>
+
+        <pre><code>&lt;h1 id="irc-log">IRC Log&lt;/h1>
+&lt;div role="log" aria-labelledby="irc-log">
+ &lt;p>[10:26] &amp;lt;Charl> ok let&#39;s test it and see if it works&lt;/p>
```

Comment:
```suggestion
 &lt;p>[10:26] <Charl> ok let&#39;s test it and see if it works&lt;/p>
```

This instance of `&lt;` was changed to `&amp;lt;` by the markdown-to-markup conversion, and that ends up being surfaced in the document. Important to have the converter fixed, or maybe just start with markup instead, to make changes easier to read. For example, this code contains `let&#39;s` instead of `let's` and `&lt;` in front of every html tag.